boileroom/models/minifold/minifold.py

```
@@ -0,0 +1,99 @@
+            Optional device specifier to pass to the backend (e.g., "cuda:0" or "cpu").
+        config : Optional[dict]
+            Optional configuration passed to the backend; if omitted an empty dict is used.
+        """
+        if config is None:
+            config = {}
+        self.config = config
+        self.device = device
+        if backend == "modal":
+            self._backend = ModalBackend(ModalMiniFold, config, device=device)
+        else:
+            raise ValueError(f"Backend {backend} not supported. MiniFold supports 'modal'.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)